src/library_pipefs.js

```diff
@@ -0,0 +1,208 @@
+mergeInto(LibraryManager.library, {
+  $PIPEFS__postset: '__ATINIT__.push(function() { PIPEFS.root = FS.mount(PIPEFS, {}, null); });',
```

Comment:
Hmm, this is actually a bit suspect: I don't think we have any type of ordering guaranteed in our JS dependencies, so calling `FS.mount()` here as a postset to PIPEFS might not guarantee that `FS.staticInit()` would have been called before. I think if it does, then it's due to 'f' < 'p' alphabetically by accident?